components/admin_console/localization_settings.jsx

```diff
@@ -31,7 +30,7 @@ export default class LocalizationSettings extends AdminSettings {
     }
 
     canSave() {
-        return this.state.availableLocales.join(',').indexOf(this.state.defaultClientLocale) !== -1 || this.state.availableLocales.length === 0;
+        return true;
```

Comment:
Error checking is now done entirely on the server side

Reply:
You can just remove the canSave function (and the binding of it) if it's just going to return true